spec/classes/agent_spec.rb

```diff
@@ -590,7 +590,7 @@
       it 'should fail' do
         expect {
           should contain_class('puppet::agent')
-        }.to raise_error(Puppet::Error,/^undefined method `to_i'/)
+        }.to raise_error(Puppet::Error,/puppet::agent::puppet_masterport is set to <invalidtype>. It should be an integer./)
```

Comment:
period `.` acts like a `*` in a regex and matches everything. They should be escaped such as `/This is how to handle a period\. It has a slash in front of it\./`